server/protocols/webadmin-cli/README.md

```diff
@@ -123,4 +151,171 @@ Show all users' name on the list.
 ### Update a user password
 Same as Create, but a user need to exist.
 
-If the user do not exist, then it will be created.
\ No newline at end of file
+If the user do not exist, then it will be created.
+
+## Manage Mailboxes
+
+### Create a mailbox
+Create a specific mailbox for a user:
+
+```
+{cli} mailbox create <username> <mailboxName>
+```
+
+Resource name username should be an existing user.
+
+Resource name mailboxName should not be empty, nor contain # & % * characters.
+
+To create nested mailboxes, for instance a work mailbox inside the INBOX mailbox, people should use the . separator. E.g: INBOX.work
+
+### Test a mailbox existence
+
+```
+{cli} mailbox exist <usernameToBeUsed> <mailboxNameToBeTested>
+```
+
+Resource name usernameToBeUsed should be an existing user 
+
+Resource name mailboxNameToBeTested should not be empty
+
+### Delete a mailbox and its children
+
+```
+{cli} mailbox delete <usernameToBeUsed> --mailbox <mailboxNameToBeDeleted>
```

Comment:
Why do we need --mailbox here?

Reply:
I make --mailbox option to put "Delete a mailbox and its children" and "Delete all mailboxes of a user" in 1 command line.

--mailbox option is not required. If there is --mailbox option, will delete only a specified mailbox; else delete all mailboxes of a user

Reply:
How about ...

```
{cli} mailbox delete <usernameToBeUsed> --mailbox <mailboxNameToBeDeleted>
{cli} mailbox delete <usernameToBeUsed> --allMailboxes
```

?